tinytodo/src/context.rs

```diff
@@ -266,12 +277,32 @@ impl AppContext {
                     AppQueryKind::GetLists(r) => self.get_lists(r),
                     AppQueryKind::AddShare(r) => self.add_share(r),
                     AppQueryKind::DeleteShare(r) => self.delete_share(r),
+                    AppQueryKind::ResetWatch => self.reset_watch(),
+                    AppQueryKind::UpdatePolicySet => self.update_policy_set(),
                 };
-                msg.sender.send(r).unwrap();
+                if let Err(e) = msg.sender.send(r) {
+                    trace!("Failed send response: {:?}", e);
+                }
             }
         }
     }
 
+    #[tracing::instrument]
```

Comment:
Remove all debugging traces etc. before merging.

Reply:
(here and elsewhere in this file)

Reply:
These are not debugging traces, these are for `tracing`, the logging library we use.